### PR TITLE
Sdit 927 charge ids

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationResource.kt [hunk 1]
```diff
@@ -155,6 +155,65 @@ class AdjudicationResource(
...
+  )
+  fun getAdjudicationCharge(
+    @Schema(description = "Adjudication number", example = "12345", required = true)
+    @PathVariable
+    adjudicationNumber: Long,
+    @Schema(description = "Charge sequence", example = "1", required = true)
+    @PathVariable
+    chargeSequence: Int,
+  ): AdjudicationResponse =
+    adjudicationService.getAdjudication(adjudicationNumber)
```
Comment (lbennett-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationResource.kt:215]: the implementation of this endpoint will follow in the next PR

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationResource.kt [hunk 2]
```diff
@@ -155,6 +155,65 @@ class AdjudicationResource(
   ): AdjudicationResponse =
     adjudicationService.getAdjudication(adjudicationNumber)
 
+  @PreAuthorize("hasRole('ROLE_NOMIS_ADJUDICATIONS')")
+  @GetMapping("/adjudications/adjudication-number/{adjudicationNumber}/charge-sequence/{chargeSequence}")
+  @ResponseStatus(HttpStatus.OK)
+  @Operation(
+    summary = "get adjudication by adjudication number",
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationResource.kt:162]: minor description change

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationResource.kt [hunk 3]
```diff
@@ -155,6 +155,65 @@ class AdjudicationResource(
...
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Adjudication does not exist",
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationResource.kt:197]: same here

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**